tests/manager/integration/license/opds/for_distributors/test_api.py

```diff
@@ -11,18 +11,74 @@
     DirectFulfillment,
     RedirectFulfillment,
 )
+from palace.manager.celery.tasks import opds_for_distributors
+from palace.manager.integration.license.opds.for_distributors.api import (
+    OPDSForDistributorsAPI,
+)
+from palace.manager.integration.license.opds.for_distributors.settings import (
+    OPDSForDistributorsSettings,
+)
 from palace.manager.integration.license.opds.requests import OAuthOpdsRequest
 from palace.manager.integration.license.overdrive.api import OverdriveAPI
-from palace.manager.sqlalchemy.constants import MediaTypes
+from palace.manager.sqlalchemy.model.collection import Collection
 from palace.manager.sqlalchemy.model.datasource import DataSource
 from palace.manager.sqlalchemy.model.identifier import Identifier
+from palace.manager.sqlalchemy.model.library import Library
 from palace.manager.sqlalchemy.model.licensing import DeliveryMechanism, RightsStatus
 from palace.manager.sqlalchemy.model.patron import Loan
 from palace.manager.sqlalchemy.model.resource import Hyperlink, Representation
 from palace.manager.util.datetime_helpers import utc_now
-from tests.manager.integration.license.opds.for_distributors.conftest import (
-    OPDSForDistributorsAPIFixture,
-)
+from tests.fixtures.database import DatabaseTransactionFixture
+from tests.fixtures.files import OPDSForDistributorsFilesFixture
+from tests.fixtures.http import MockHttpClientFixture
+from tests.fixtures.work import WorkIdPolicyQueuePresentationRecalculationFixture
+
+
+class OPDSForDistributorsAPIFixture:
```

Comment:
This is just moved from the `conftest.py` file, since the fixture is no longer shared with with the importer tests.